<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_FieldMap_R2L">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>492</width>
      <height>345</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="CoachResource.qrc">:/Image/Config/R2LFullField.bmp</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_FieldMap_L2R">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>492</width>
      <height>345</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="CoachResource.qrc">:/Image/Config/L2RFullField.bmp</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_1_Online">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>411</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_1_IP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_1_LapBat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_1_AgBat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_2_Online">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>411</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_2_LapBat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_2_IP">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_2_AgBat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_3_Online">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>411</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_3_LapBat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_3_IP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_3_AgBat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_4_Online">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>411</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_4_LapBat">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_4_IP">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_4_AgBat">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_5_Online">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>411</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_5_LapBat">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_5_IP">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_5_AgBat">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>501</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_1_CurStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>471</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_2_CurStatus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>471</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_3_CurStatus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>471</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_4_CurStatus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>471</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_5_CurStatus">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>471</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>409</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Agent 6:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_6_LapBat">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_6_AgBat">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_6_IP">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>439</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Agent_6_CurStatus">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Agent_6_Online">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>21</y>
      <width>181</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sideTag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>sideTag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RefIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectREF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connet REF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_KickOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m_KickOff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FreeKick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m_FreeKick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CornerKick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m_CornerKick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GoalKick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m_GoalKick</string>
    </property>
   </widget>
   <widget class="FieldMap" name="label_FieldMap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>492</width>
      <height>345</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Status">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionEmergency_Stop"/>
    <addaction name="actionChange_Side"/>
    <addaction name="actionAttack_Action"/>
    <addaction name="actionStop_One_Agent"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEmergency_Stop">
   <property name="text">
    <string>Emergency Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionChange_Side">
   <property name="text">
    <string>Change Side</string>
   </property>
  </action>
  <action name="actionAttack_Action">
   <property name="text">
    <string>Attack Action</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionStop_One_Agent">
   <property name="text">
    <string>Stop One Agent</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FieldMap</class>
   <extends>QLabel</extends>
   <header location="global">WidgetsPlus/FieldMap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CoachResource.qrc"/>
 </resources>
 <connections/>
</ui>
